DigitalLearningSolutions.Data/Models/MultiPageFormData/AddNewCentreCourse/CourseTutorialData.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace DigitalLearningSolutions.Data.Models.MultiPageFormData.AddNewCentreCourse
+{
+    public class CourseTutorialData
+    {
+        // 'Unused' constructor required for JsonConvert
+        public CourseTutorialData(){}

```

Comment:
Yeah I think we're just going to have to ignore this one existing (like we do elsewhere). Changing it to nullable will cause a cascade of further changes making it a nullable string where it isn't actually nullable.

Reply:
Not really sure, it'll get rid of the warnings, which I guess is nice. We're already at the point where I've disabled showing warnings in the Error List because they're nearly all useless ones like this.

If we do the `= null!` it'll just throw null reference exceptions when someone forgets to set the property before accessing it, so I guess it's probably fine?

Reply:
We've decided to leave these because it won't realistically be of much benefit to the project.